http-source/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -4,7 +4,7 @@
       "name": "pathPattern",
       "type": "java.lang.String",
       "description": "An Ant-Style pattern to determine which http requests will be captured",
-      "defaultValue": "/messages"
+      "defaultValue": "/"
```

Comment:
I suppose Docs should be fixed to address this change.
And, of course, it would be better to do that during this PR.
I mean something like:
```